src/Pasajero.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Mibici;
+
+class Pasajero implements PasajeroInterface {
+    protected $dni;
+    protected $saldo;
+
+    function __construct($doc,$dinero) {
+        $this->dni=$doc;
+        $this->saldo=$dinero;
```

Comment:
Saquemos saldo. Enfoquense en el dni. Además usen nombres de variables consistentes. `$this->dni = $dni`